### PR TITLE
[CCM] add AWS metric and tag source documentation

### PR DESCRIPTION
**Pull request by enemyghost**
### What does this PR do? What is the motivation?
- adding new metrics
- improving metric descriptions + providing advice
- adding tag sources

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/cloud_cost_management/aws.md
Comment (glebec) [inline content/en/cloud_cost_management/aws.md:143]: > accrual date

Is the reader expected to know what "accrual date" means? Is it the date that a payment was applied?

Comment (glebec) [reply content/en/cloud_cost_management/aws.md:143]: The sentence starting "Not all one-time fees..." feels like it could be a second short paragraph.

Comment (glebec) [reply content/en/cloud_cost_management/aws.md:143]: What is an "RI Fee"?

**Comment by glebec** [comment]
I think this helps to clarify a lot; one thing I noticed is that the "on-demand" cost isn't explained until the Example section. Maybe it could get a short explanation before then?

**Review by glebec — APPROVED** [review]

File: content/en/cloud_cost_management/aws.md
Comment (urseberry) [inline content/en/cloud_cost_management/aws.md:145]:
```suggestion
On-demand costs do not accurately represent what you pay AWS. Do not use this metric to estimate your actual costs.
```

File: content/en/cloud_cost_management/aws.md
Comment (urseberry) [inline content/en/cloud_cost_management/aws.md:147]:
```suggestion
### Amortized and unblended costs
```

File: content/en/cloud_cost_management/aws.md
Comment (urseberry) [inline content/en/cloud_cost_management/aws.md:148]:
```suggestion
**Amortized** cost metrics distribute some pre-payments (such as up-front savings plan fees) across the entire discount term. Another name for amortized cost metrics is _accrual basis_. As savings plans are applied to covered usage, the costs for that savings plan appear as usage charges. The unused portion appears as a fee.
```

File: content/en/cloud_cost_management/aws.md
Comment (urseberry) [inline content/en/cloud_cost_management/aws.md:152]:
```suggestion
**Unblended** cost metrics show all charges on the date that they are incurred. Another name for unblended cost metrics is _cash basis_. These metrics match the AWS invoice exactly.
```

**Review by urseberry — COMMENTED** [review]
I am part way through reviewing - will finish later today or tomorrow.

File: content/en/cloud_cost_management/aws.md
Comment (enemyghost) [reply content/en/cloud_cost_management/aws.md:148]: Not sure about this one, since we actually don't call this anywhere in our own product... it's more like "this type of cost reporting is also known as accrual basis / cash basis".

File: content/en/cloud_cost_management/aws.md
Comment (guilhermevrs) [inline content/en/cloud_cost_management/aws.md:157]: **💭 thought:** ‏Perhaps we should refrain of talking about keywords and suffixes. I believe our overall goal is to hide the name of the metrics (once we migrate the new searchbar to dashboards).
If we don't have a better solution now, we can revisit that later

Comment (enemyghost) [reply content/en/cloud_cost_management/aws.md:157]: great point, I will try to revise this. I will leave the full metric names in the Metric Summary table for now, and we can modify that when they're no longer visible if we want.

Though, I'm guessing you will still see them in string-query edit mode?

File: content/en/cloud_cost_management/aws.md
Comment (urseberry) [reply content/en/cloud_cost_management/aws.md]: Got it. Thanks for explaining.

File: content/en/cloud_cost_management/aws.md
Comment (urseberry) [inline content/en/cloud_cost_management/aws.md:228]:
```suggestion
When you enable the Datadog AWS integration, Datadog automatically collects resource tags for most AWS resources. These tags are applied to all costs found in the CUR for a given resource. Resource tags are retrieved regularly and are applied to cost data starting from the day they are created or modified. Historical tag values are not overwritten when tags change.
```

File: content/en/cloud_cost_management/aws.md
Comment (enemyghost) [reply content/en/cloud_cost_management/aws.md:168]: Your interpretation is correct, though i was trying to skip over the complexity of "purchased at a cost of $X". Updated to remove - does it make sense?

File: content/en/cloud_cost_management/aws.md
Comment (urseberry) [inline content/en/cloud_cost_management/aws.md:233]:
```suggestion
AWS Organizations support [user-defined tags][14] on organizational units and accounts. Datadog automatically fetches and applies these tags to cost data. Account tags are applied to all usage associated with those accounts. Organization tags are applied to all billing data for the matching payer account.
```

File: content/en/cloud_cost_management/aws.md
Comment (enemyghost) [reply content/en/cloud_cost_management/aws.md:157]: @urseberry  Tried to reword this a bit. Had some feedback that "non-net" is not really a term we want to introduce since it's a bit confusing. Wdyt?

File: content/en/cloud_cost_management/aws.md
Comment (urseberry) [inline content/en/cloud_cost_management/aws.md:239]:
```suggestion
AWS integration tags are tags set on the AWS Integration tile in the Datadog integrations page. They are applied to all costs found in the CUR for the associated AWS account.
```

File: content/en/cloud_cost_management/aws.md
Comment (urseberry) [inline content/en/cloud_cost_management/aws.md:299]:
```suggestion
Container cost allocation adds tags from the workloads incurring cost. Examples include tags from Kubernetes pods and nodes and ECS tasks and containers.
```

File: content/en/cloud_cost_management/aws.md
Comment (urseberry) [inline content/en/cloud_cost_management/aws.md:301]:
```suggestion
_Requires [container cost allocation][11], and applies only to `shared.resources.allocated` metrics._
```

**Review by urseberry — COMMENTED** [review]
Whew, all done with my text review. Nice job on this - I appreciate your precise language. Describing the details of these different cost calculations is a difficult task.

I'll go back and do one more pass to check the links, which I just noticed, but I assume those are probably going to be fine.

File: content/en/cloud_cost_management/aws.md
Comment (urseberry) [reply content/en/cloud_cost_management/aws.md:168]: Yes, it makes sense, thanks!

File: content/en/cloud_cost_management/aws.md
Comment (urseberry) [reply content/en/cloud_cost_management/aws.md:157]: Yes, your latest wording looks great.

**Comment by enemyghost** [comment]
Thanks @urseberry for the thorough review, really appreciate your edits. I think at least one more subject matter expert will review the content, then I'll get your final sign off.

File: content/en/cloud_cost_management/aws.md
Comment (cnharris10) [inline content/en/cloud_cost_management/aws.md:214]:
```suggestion
To ensure consistency, Datadog normalizes tag keys using underscores and lower case. For example, the CUR column `lineItem/ResourceId` maps to the tag key `line_item/resource_id`. Tag values are generally unmodified - maintaining exact casing and most special characters.
```

File: content/en/cloud_cost_management/aws.md
Comment (cnharris10) [inline content/en/cloud_cost_management/aws.md:220]:
```suggestion
|lineItem/ResourceId|i-12345678a9b12cd3e|line_item/resource_id:i-12345678a9b12cd3e|
```

**Review by cnharris10 — CHANGES_REQUESTED** [review]
Let me know your thoughts on these suggestions.

File: content/en/cloud_cost_management/aws.md
Comment (enemyghost) [reply content/en/cloud_cost_management/aws.md:157]: i think `some` is still a good edit, though I did initially mean resource id attribution.

File: content/en/cloud_cost_management/aws.md
Comment (enemyghost) [reply content/en/cloud_cost_management/aws.md:148]: agree on both counts 
- i will switch to "cost basis"
- reworded your edit a bit, lmk if it makes sense.

Comment (enemyghost) [reply content/en/cloud_cost_management/aws.md:148]: with this edit, maybe i will delete the next paragraph, which I think is now confusing after this description.

File: content/en/cloud_cost_management/aws.md
Comment (enemyghost) [reply content/en/cloud_cost_management/aws.md:145]: I didn't want to introduce the term "unblended" here yet, and I think this edited sentence is kind of confusing. In this section, I try to provide light guidance, not a complete description with all caveats. Are you concerned about saying "they do not reflect actual costs" when in some cases they do?

What about something like:

In most cases, on-demand costs are not a reliable source to estimate actual costs.

File: content/en/cloud_cost_management/aws.md
Comment (enemyghost) [inline content/en/cloud_cost_management/aws.md:151]:
```suggestion
```

File: content/en/cloud_cost_management/aws.md
Comment (enemyghost) [inline content/en/cloud_cost_management/aws.md:152]:
```suggestion
**Unblended** cost metrics show all charges on the date that they are incurred. This is also called _cost basis_. These metrics match the AWS invoice exactly after a month is finalized.
```

**Review by urseberry — APPROVED** [review]
Looks like this is good to merge after @cnharris10 approves?

**Review by cnharris10 — APPROVED** [review]
I still find the Cost metric summary table hard to follow, particularly since the `Usage` and `Discount` columns show effective costs, but the `Savings Plan` column does not.  I won't block on this as long as others feel differently.

**Pull request merged**
